elements/pl-matrix-component-input/pl-matrix-component-input.mustache

```diff
@@ -92,3 +96,13 @@ No symbolic expressions (those that involve fractions, square roots, variables,
 {{#shortformat}}
 {{#relabs}}(rtol={{rtol}}, atol={{atol}}){{/relabs}}{{#sigfig}}({{digits}} significant figures){{/sigfig}}{{#decdig}}({{digits}} digits after decimal){{/decdig}}
 {{/shortformat}}
+
+{{#format_error}}
+    <p> Invalid format: one or more entries of the submitted answer could not be interpreted as a decimal {{#complex}}or complex{{/complex}} number.  No symbolic input or expressions are allowed.</p>
+    <p> Example <i>valid</i> inputs: <code class="user-output">5</code> <code class="user-output">-17</code> <code class="user-output">4.25e7</code> {{#complex}}<code class="user-output">3.6i</code>{{/complex}} </p>
+    <p> Example <i>invalid</i> inputs: <code class="user-output-invalid">6+9</code> <code class="user-output-invalid">4.25*10^7</code> <code class="user-output-invalid">3*pi</code> {{^complex}}<code class="user-output-invalid">3.6i</code>{{/complex}} </p>
+    <hr>
+    <small>
+        <p> Each entry of the submitted answer must be formatted as an <a href="https://en.wikipedia.org/wiki/Double-precision_floating-point_format">IEEE 754 double-precision floating-point</a> {{#complex}}or complex{{/complex}} number.  These numbers lie roughly in the range of -1.79 &times; 10<sup>308</sup> to 1.79 &times; 10<sup>308</sup>.</p>
```

Comment:
Maybe best not to link to public websites, because access is blocked inside the CBTF?